config/test/msw/handlers.ts

```diff
@@ -0,0 +1,62 @@
+import { http, HttpResponse } from 'msw'
```

Comment:
The handlers are now shared between web and mobile. The idea for this handlers would be to have default data for the endpoints we use across the project. We should combine this with the fake builders.

note how every response endpoint specifies the type of the returned JSON. With this whenever a type in the store package change we can be sure that we also need to update a test as the type would become invalid.